openlibrary/templates/covers/add.html

```diff
@@ -50,10 +50,26 @@
     </div>
     <div class="input">
         <input type="file" name="file" id="coverFile" value="" accept=".jpg, .jpeg, .gif, .png" required/>
-        <button type="submit" class="cta-btn cta-btn--vanilla">$_("Upload")</button>
+        <button id='uploadButton1' type="submit" class="cta-btn cta-btn--vanilla">$_("Upload")</button>
```

Comment:
This is unused
```suggestion
        <button type="submit" class="cta-btn cta-btn--vanilla">$_("Upload")</button>
```

Reply:
I have removed the id='uploadButton1'. For better readability, rather than naming the other button as uploadButton2 or uploadButton, I have named in uploadButtonPaste